shell.nix

```diff
@@ -20,7 +20,7 @@
     go_1_21
     gopls
     delve
-    golangci-lint
+    (golangci-lint.override { buildGoModule = buildGo121Module; })
```

Comment:
This seems really bizarre. We can't rig up something derived from the go.mod?

Hopefully this goes away in the future, when the go tooling can download other versions of itself. :shrug: